src/ts_catalog/compression_chunk_size.c

```diff
@@ -34,6 +34,7 @@ ts_compression_chunk_size_delete(int32 uncompressed_chunk_id)
 	{
 		TupleInfo *ti = ts_scan_iterator_tuple_info(&iterator);
 		ts_catalog_delete_tid(ti->scanrel, ts_scanner_get_tuple_tid(ti));
+		count++;
```

Comment:
The return value was unused and is still unused, right? Maybe we should remove it instead?

This is an example of changes independent of TAM that we could review and merge separately.